src/views/Swap/components/SwapCommitButton.tsx

```diff
@@ -116,7 +117,7 @@ export default function SwapCommitButton({
         setSwapState({
           attemptingTxn: false,
           tradeToConfirm,
-          swapErrorMessage: error.message,
+          swapErrorMessage: getErrorReasonFromTransactionError(error),
```

Comment:
can you help check if UserRejectedRequestError
I think we shouldn't reset the whole swap state there

Reply:
Ah right. I mean we don't need to show transaction reject?
Just revert to no tx attempting